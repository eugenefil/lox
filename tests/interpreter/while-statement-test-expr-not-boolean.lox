while 1 {
    y;
}
