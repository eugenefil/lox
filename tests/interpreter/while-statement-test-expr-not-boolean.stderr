error: expected 'Bool', got 'Number'
 --> $DIR/while-statement-test-expr-not-boolean.lox:1:7
  |
1 | while 1 {
  |       ^
